src/high-level/src/KinDynComputations.cpp

```diff
@@ -1536,6 +1536,67 @@ Vector3 KinDynComputations::getCenterOfMassBiasAcc()
     return comBiasAcc;
 }
 
+Vector6 KinDynComputations::getLinearAngularMomentumBiasAcc()
+{
+    this->computeRawMassMatrixAndTotalMomentum();
+    this->computeBiasAccFwdKinematics();
+
+    // We compute the bias of the center of mass from the bias of the total momentum derivative
+    Wrench totalMomentumBiasInInertialInertial;
+    ComputeLinearAndAngularMomentumDerivativeBias(pimpl->m_robot_model,
+                                                  pimpl->m_linkPos,
+                                                  pimpl->m_linkVel,
+                                                  pimpl->m_linkBiasAcc,
+                                                  totalMomentumBiasInInertialInertial);
+
+    // The total momentum is written in the (inertial,inertial) frame
+    // To get the com velocity, acceleration, we need to write it in the (com,inertial) frame
+    // For the velocity we need just to multiply it for the adjoint, for the acceleration
+    // we need also to account for the derivative of the adjoint term
```

Comment:
This comment see to be a copy-paste left over from getCenterOfMassBiasAcc, and it does not apply anymore here I guess?